AcademyApp/Extensions/UIColor.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  UIColor.swift
+//  AcademyApp
+//
+//  Created by Peter Sevcik on 13/06/2019.
+//  Copyright © 2019 Peter Sevcik. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIColor
```

Comment:
Naming conventions for extension are usually 
UIColor+Feature like UIView+RoundedBorders
or UIColorExtensions

Reply:
Renamed UIColor to UIColorExtensions